README.md

```diff
@@ -648,16 +648,16 @@ The verification information consists of the following information:
    </td>
   </tr>
   <tr valign="top">
-   <td>Dispensations
+   <td>Dispenses
```

Comment:
Shouldn't this be "Events" (when looking at the example response?

Reply:
Changed